Doc/library/test.rst

```diff
@@ -334,12 +351,6 @@ The :mod:`test.support` module defines the following constants:
    Set to :data:`sys.maxsize` for big memory tests.
 
 
-.. data:: max_memuse
```

Comment:
I think it's better to keep this documented for symmetry with `real_max_memuse`.